infra/common/policy/policy-diagnostic-settings-assignment.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
bicep params?

Reply:
We have an issue with the bicep param with json. So we need to find a fix.